src/api.jl

```diff
@@ -485,7 +485,6 @@ end
 
 function h5p_get_driver_info(plist_id)
     var"#status#" = ccall((:H5Pget_driver_info, libhdf5), Ptr{Cvoid}, (Hid,), plist_id)
-    var"#status#" < 0 && error("Error getting driver info")
```

Comment:
Is the CNull error handling here not being picked up?

Reply:
No, that's on purpose — [`H5Pget_driver_info`](https://support.hdfgroup.org/HDF5/doc/RM/H5P/H5Pget_driver_info.htm) doesn't error because it's valid for the driver to not have extra info. That's the reason I had to allow for a no-error case in the generator (setting `err = nothing` by default in the macro definition).

Reply:
Oh I see err has to be set and the return type has to be Ptr for that case to be generated